packages/bindings/src/sudo.rs

```diff
@@ -20,6 +21,12 @@ pub enum TgradeSudoMsg {
     /// which will be used to change the validator set.
     EndWithValidatorUpdate {},
     PrivilegeChange(PrivilegeChangeMsg),
+    /// This will export contract state. Requires `StateExporterImporter` privilege.
+    Export {},
+    /// This will import contract state. Requires `StateExporterImporter` privilege.
+    Import {
+        import: ExportImport<S>,
```

Comment:
This results in: `{"import":{"import":{"version":"0.8.1","state":{}}}}`

Reply:
Simplified it.